docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -684,33 +698,50 @@ version 3 of link:https://search.maven.org/artifact/com.azure.spring/azure-sprin
-| *azure.keyvault*.client-key              | *spring.cloud.azure.keyvault.secret*.property-sources[n].credential.client-secret
-| *azure.keyvault*.enabled                 | *spring.cloud.azure.keyvault.secret*.enabled and *spring.cloud.azure.keyvault.secret*.property-source-enabled
-| *azure.keyvault*.order                   | Not supported anymore. Use the order in property-sources[n] instead.
-| *azure.keyvault*.refresh-interval        | *spring.cloud.azure.keyvault.secret*.property-sources[n].refresh-interval
-| *azure.keyvault*.secret-keys             | *spring.cloud.azure.keyvault.secret*.property-sources[n].secret-keys
-| *azure.keyvault*.tenant-id               | *spring.cloud.azure.keyvault.secret*.property-sources[n].profile.tenant-id
-| *azure.keyvault*.uri                     | *spring.cloud.azure.keyvault.secret*.property-sources[n].endpoint
+
+| *azure.keyvault*.case-sensitive-keys
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].case-sensitive
+
+| *azure.keyvault*.certificate-password
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].credential.client-certificate-password
+
+| *azure.keyvault*.certificate-path
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].credential.client-certificate-path
+
+| *azure.keyvault*.client-id
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].credential.client-id
+
+| *azure.keyvault*.client-key
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].credential.client-secret
+
+| *azure.keyvault*.enabled
+| *spring.cloud.azure.keyvault.secret*.enabled and *spring.cloud.azure.keyvault.secret*.property-source-enabled
+
+| *azure.keyvault*.refresh-interval
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].refresh-interval
+
+| *azure.keyvault*.secret-keys
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].secret-keys
+
+| *azure.keyvault*.tenant-id
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].profile.tenant-id
+
+| *azure.keyvault*.uri
+| *spring.cloud.azure.keyvault.secret*.property-sources[n].endpoint
 |===
 
 .Removed properties from spring-cloud-azure-starter-keyvault-secrets
-[options="header"]
-|===
-| Removed property
-| azure.keyvault.allow-telemetry
-| azure.keyvault.order
-|===
+* azure.keyvault.allow-telemetry
+* azure.keyvault.order
+
 
+.The following points you should pay your attention to:
 . All configuration property names changed the prefix from `azure.keyvault` to `spring.cloud.azure.keyvault.secret`.
 . `spring.cloud.azure.keyvault.secret.enabled` is used to enable all Key Vault Secret features, include configure Key Vault secret client beans(like `SecretClient` and `SecretAsyncClient`) and add `KeyVaultPropertySource` in `ConfigurableEnvironment`.
 . `spring.cloud.azure.keyvault.secret.property-source-enabled` is used to enable all `KeyVaultPropertySource`. It will take effect only when `spring.cloud.azure.keyvault.secret.enabled=true`.
```

Comment:
> . `spring.cloud.azure.keyvault.secret.property-source-enabled` is used to enable all `KeyVaultPropertySource`. It will take effect only when `spring.cloud.azure.keyvault.secret.enabled=true`.

Change to 

> . `spring.cloud.azure.keyvault.secret.property-source-enabled` is used to enable all `KeyVaultPropertySource`.

Reply:
yes, updated